pkg/edit/editor_test.go

```diff
@@ -37,6 +37,36 @@ func TestEditor_DoesCommandWithLeadingSpaceToHistory(t *testing.T) {
 	testCommands(t, f.Store /* no commands */)
 }
 
+func TestEditor_TestBeforeAddCmd(t *testing.T) {
+	cases := []struct {
+		name string
+		rc   string
+		in   string
+		exp  []string
+	}{
+		{"nonCallableItem", "edit:add-cmd-filters = [$false]", "echo\n", []string{"echo"}},
```

Comment:
Write out the name as words and make them more complete; this is intended for human consumption. For example, this name is fine, but can be spread out as `"non callable item"`